README.md

```diff
@@ -1,76 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome books
 
-> One paragraph statement about the project.
+> This is a javascript book list application
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./app-screenshot.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- HTML
+- SASS
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
```

Comment:
- [ ] I would recommend adding all the steps required to run the project locally. 

```

$ git clone https://github.com/Leboroz/awesome-books.git

$ cd awesome-books

$ open index.html
```